README.md

```diff
@@ -27,7 +27,39 @@ Notebooks uses AI Platform version 1.14, we had to code in Python 3.5 (no
 f-strings or type hinting)
 
 ### Jupyter via remote execution into AI Platform Notebook Instance
-Coming Soon - Some documentation already complete by team
+Prerequisites: 
+1. You must have Anaconda/Jupyter Notebook installed 
+2. If using a windows machine, you must have PuTTY installed
+3. You must have Cloud SDK installed: https://cloud.google.com/sdk/install
+
+Helpful Links: 
+https://cloud.google.com/ai-platform/notebooks/docs/ssh-access
+
+Instructions:
+1. On the AI Platform Notebook Instance you created,
+open your VM Instance details.
+On the dropdown for Remote access, select "view gcloud command"
+Example: 
+gcloud compute --project "project-name" ssh --zone "us-east1-d" "instance-name"
```

Comment:
One more question on this line - is specifying a zone required for instances that are already created with a zone? I might be forgetting something here, but curious.